### DISCUSSION TITLE
How do I fix the whole site crashing when I connect to my domain?

### DISCUSSION THREAD

**Discussion by sxadxsx** [upptime/upptime#594] [Q&A] [answered]
How do I fix the whole site crashing when I connect to my domain?
![Screenshot_20220614-224006_Chrome](https://user-images.githubusercontent.com/107471917/173605752-0f0da41f-4465-4bb3-80c7-20943d96072d.jpg)

**Comment by AnandChowdhary** [comment, marked as answer]
It looks like you have not added the CNAME here: https://github.com/sxadxsx/upptime/blob/master/.upptimerc.yml#L28. Or is this a different repository? Usually it's a configuration issue when the CSS doesn't load. :)

**Reply by sxadxsx** [reply]
Thanks
